benchmarks/examples/sombrero/sombrero.py

```diff
@@ -100,6 +96,19 @@ class SombreroBenchmark(SpackTest):
         },
     }
 
+    def __init__(self):
```

Comment:
I tried it without but it just skips the file because of the following: `reframe.core.exceptions.ReframeSyntaxError: accessing a test parameter from the class body is disallowed`. I took the `__init__` idea from the IMB benchmarks.